starter/schemaorg/Makefile

```diff
@@ -91,13 +99,13 @@ custom_train_nlu_flags ?=
 .PHONY: all train evaluate
 .SECONDARY:
 
-models/paraphraser-bart:
-	mkdir models
-	wget --no-verbose https://almond-static.stanford.edu/test-data/paraphraser-bart.tar.xz
-	tar -C models -xvf paraphraser-bart.tar.xz
+models/paraphraser-bart-large-speedup-megabatch-5m:
+	mkdir -p models
+	wget --no-verbose https://almond-static.stanford.edu/test-data/paraphraser-bart-large-speedup-megabatch-5m.tar.xz
+	tar -C models -xvf paraphraser-bart-large-speedup-megabatch-5m.tar.xz
```

Comment:
Do we need to distribute both paraphraser models? Can we kill the old one?

Reply:
you can kill the other one

Reply:
What i mean is overwrite the old model and keep the old URL, but point to the new model.